plugins/json-multisheet-extractor/jest.config.cjs

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  testEnvironment: 'node',
+
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  setupFiles: ['../../test/dotenv-config.js'],
+  setupFilesAfterEnv: [
+    '../../test/betterConsoleLog.js',
+    '../../test/unit.cleanup.js',
+  ],
+  testTimeout: 60_000,
+  globalSetup: '../../test/setup-global.js',
+  forceExit: true,
+  passWithNoTests: true,
+}
```

Comment:
AI bot suggestion around code coverage requirements https://github.com/FlatFilers/flatfile-plugins/pull/693/files#r1826003527

Sounds great on paper but I don't want to add a new PR constraint when the repo isn't enforcing/using code coverage anywhere else. Great change for a separate PR